radar/languages-frameworks.js

```diff
@@ -122,32 +128,26 @@ const content = [
     requiring payment.
   </li>
   <li>
-    <strong>Integrated Debugging Tools:</strong> Built-in debugging capabilities simplify troubleshooting during test
+    <strong>Integrated Debugging Tools:</strong>Built-in debugging capabilities simplify troubleshooting during
     development.
   </li>
+  <li>
+    <strong>Potential bottleneck:</strong> Parallelism of tests inside a single CI/CD runner can make a testrun slower. This can be solved by parallelizing the tests over multiple runners.
```

Comment:
Sharding over meerdere runners is inderdaad positief, maar je begint met de woorden Potential Bottleneck? Dat is wat maf.

Suggestie:

```suggestion
    <strong>Multiple options for parallelism:</strong> Cypress offers features to parallelize tests on a single CI/CD runner as well as multiple runners. 
```

Is bovenstaande een betaalde feature?